java-connectors/kafka-connect-azure-eventhubs/src/main/java/io/lenses/java/streamreactor/connect/azure/eventhubs/reading/EventProcessorBlobCheckpointStoreSample.java

```diff
@@ -0,0 +1,71 @@
+//package io.lenses.java.streamreactor.connect.azure.eventhubs.reading;
```

Comment:
Why not remove this since it's not used?

Reply:
my bad, thanks for checking! deleted now